tests/purs/bundle/3551/ModuleWithDeadCode.js

```diff
@@ -1,10 +1,8 @@
-"use strict";
-
-var fs = require('fs');
+import * as fs from 'fs';
```

Comment:
It would be more idiomatic to write `import fs from 'fs'` or `import { readFileSync } from 'fs'` here but `purs bundle` does not support these syntaxes yet.